contrib/pyln-spec/bolt2/pyln/spec/bolt2/csv.py

```diff
@@ -39,6 +41,8 @@
     "msgdata,accept_channel,tlvs,accept_channel_tlvs,",
     "tlvtype,accept_channel_tlvs,upfront_shutdown_script,0",
     "tlvdata,accept_channel_tlvs,upfront_shutdown_script,shutdown_scriptpubkey,byte,...",
+    "tlvtype,accept_channel_tlvs,channel_type,1",
+    "tlvdata,accept_channel_tlvs,channel_type,type,byte,...",
```

Comment:
This is a duplicate of lines 24-25, since this is derived from the spec we'll likely have to look upstream.

Reply:
No, one is open, other is accept...

Reply:
I fall into the same reading mistake 📃